GoogleDataTransport/GDTCORLibrary/GDTCORUploadCoordinator.m

```diff
@@ -168,7 +168,7 @@ - (void)appWillBackground:(GDTCORApplication *)app {
 }
 
 - (void)appWillTerminate:(GDTCORApplication *)application {
-  dispatch_sync(_coordinationQueue, ^{
+  dispatch_async(_coordinationQueue, ^{
```

Comment:
I'm trying to think through what could possibly differ with the switch to async. Do you think it's possible for the `[self stopTimer]` to _not_ get called with the async dispatch in some instances where it _is_ called for the sync dispatch?

Reply:
My conjecture is the incidence of `stopTimer` not being called before app termination will increase because with this change app termination will no longer wait for this block to execute. But there existed this possibility beforehand of the watchdog killing the app before `stopTimer` executed anyway.

What happens if `stopTimer` isn't called before app shutdown?

Reply:
Discussed offline. It shouldn't be an issue if stopTimer is not called.